### PR TITLE
Automated cherry pick of #3107: Update coreos stack to reduce CVEs

### PR DESCRIPTION
**Pull request by rene-dekker**
Cherry pick of #3107 on master.

#3107: Update coreos stack to reduce CVEs

### CODE REVIEW

**Review by Behnam-Shobiri — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Pull request merged**
